lib/note-toolbar/index.tsx

```diff
@@ -25,6 +28,7 @@ type StateProps = {
 
 type DispatchProps = {
   deleteNoteForever: () => any;
+  onNewNote: () => any;
```

Comment:
while common, and common historically in this repo, the `onBlahBlah` naming can be verbose beyond value. our types ensure that something is a function or a value, so let's consider naming the prop more directly, something like `createNote` or `newNote` or `createNote` or `createNewNote`

Reply:
Thanks Dennis! I've updated the naming now.